src/help/help_topic_generators.cpp

```diff
@@ -324,6 +324,9 @@ std::string unit_topic_generator::operator()() const {
 		sz = 400;
 	}
 
+	// otherwise scaling down (SCALE_INTO below) and then scaling back up due to the pixel multiplier leads to ugly results
```

Comment:
This "otherwise" appears to be referring back to a nonexistent antecedent. Did you miss part of the comment?

Reply:
Clarified the comment.

Reply:
Better,  but "without" is one word.